book/_setup.qmd

```diff
@@ -2,8 +2,9 @@
 library("data.table")
 library("mlr3")
 library("mlr3book")
-requireNamespace("bbotk")
+library("bbotk")
 library("mlr3verse")
+library("mlr3mbo")
```

Comment:
This shouldn't be in the general setup file. Please move to start of this chapter instead